src/view/element.js

```diff
@@ -788,3 +819,14 @@ function normalize( nodes ) {
 			return typeof node == 'string' ? new Text( node ) : node;
 		} );
 }
+
+// Returns string representation of povided map in following format:
+//
+//		"mapKey1=value1,mapKey2=value2,mapKey3=value3"
+//
+// NOTE: All map keys should be strings. Key-value pairs will be sorted.
+//
+// @returns {String}
+function mapToSortedString( map ) {
+	return Array.from( map ).map( i => i[ 0 ] + '=' + i[ 1 ] ).sort().join( ',' );
```

Comment:
Or nevermind. In fact, it will be less efficient, and we use these methods very often. To make it in the propr way we should save styles, classes, and attributes in the ordered collection, but it's not a good ticket for it.